packages/webamp-modern-2/src/skin/Bitmap.ts

```diff
@@ -53,7 +53,7 @@ export default class Bitmap {
   }
 
   getId() {
-    return this._id;
+    return this._id || "";
```

Comment:
When does this happen? This seems a bit dangerous if it can be avoided.

Reply:
I couldn't found much, it was many problem there, maybe has been solved.
but now, I can only find this:
```
_setCssVars() {
    const cssRules = [];
    const bitmapFonts: BitmapFont[] = this._fonts.filter(
      (font) => font instanceof BitmapFont && !font.useExternalBitmap()
    ) as BitmapFont[];
    for (const bitmap of [...this._bitmaps, ...bitmapFonts]) {
      const img = bitmap.getImg();
      if (!img) {
        console.warn(`Bitmap/font ${bitmap.getId()} has no img!`);
        continue;
      }
```
:p

Reply:
Okay, this is a real error: `guiObj.ts` function findobject().
```
findobject(id) {
    if (id.toLowerCase() == this.getId().toLowerCase())
      return this;
```
this.getId() <---- error here !